### PR TITLE
STUD-216: connect send message

### PR DESCRIPTION
**Pull request by yasfahm**
connect websocket

### CODE REVIEW

File: src/screens/Classes.js [hunk 1]
```diff
@@ -7,18 +7,48 @@ import theme from '../styles/theme.style.js';
...
+    var ws = new WebSocket('ws://real.encs.concordia.ca/chat/chat/1');
+    this.setState({wso: ws})
+    ws.onmessage = (e) => {
+      // a message was received
+      console.log(e.data);
+    };
+    ws.onclose = () => {
+      console.log('disconnected');
+      this.connectOnLoad();
+  };
```
Comment (nouj) [inline src/screens/Classes.js:38]: Is there a tab missing on this line?

Comment (nouj) [inline src/screens/Classes.js]: Is there a missing indentation?

File: src/screens/Classes.js [hunk 2]
```diff
@@ -7,18 +7,48 @@ import theme from '../styles/theme.style.js';
...
+  }
+
+  onSendMessage(m) {
+    this.state.wso.send(m)
+    // ws.send(m)
   }
 
-  // Write functions here
```
Comment (nouj) [inline src/screens/Classes.js:46]: Do we need two empty lines between onSendMessage() and render()?

File: src/screens/Classes.js (hunk 1 above)
Comment (yasfahm) [reply src/screens/Classes.js]: fixed!

**Review by nouj — APPROVED** [review]

**Comment by nuzurie** [comment]
Feature redesigned and finished.

**Pull request closed without merging**